app/coffee/Features/Authentication/AuthenticationController.coffee

```diff
@@ -157,21 +157,23 @@ module.exports = AuthenticationController =
 		return isValid
 
 	_redirectToLoginOrRegisterPage: (req, res)->
-		if req.query.zipUrl? or req.query.project_name?
+		if (req.query.zipUrl? or
+			  req.query.project_name? or
+			  req.path == '/user/subscription/new')
```

Comment:
This ties in with the removal of baseUrl from the SubscriptionController. It used to direct specifically to the `register` page for the new subs, and the inclusion of `/user/subscription/new` does the same here.